src/osemgrep/language_server/Test_LS_e2e.ml

```diff
@@ -221,14 +237,28 @@ let receive_request (info : info) : Request.t Lwt.t =
 let git_tmp_path () =
   Testutil_files.with_tempdir ~persist:true (fun dir ->
       let dir = Fpath.to_string dir in
+      (* This chdir does nothing in the JSCaml code, because the OCaml path is
+         only simulated, in the Javascript environment.
+         Hence, the liberal uses of `-C` everywhere.
+         This might still be wrong, though.
```

Comment:
I like using -C instead of trying to handle it ourselves, feels less error prone